src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -96,11 +96,22 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         {
             NodeModel node = null;
 
+            String typeName = null;
+            String functionName = null;
```

Comment:
I would usually recommend to use `String.Empty` because it is less likely to cause crashes. But I saw you have wrapped string null check lots of places which is also good.